src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/integrations/utbetaling/UtbetalingController.kt

```diff
@@ -21,12 +21,13 @@ class UtbetalingController(
     @GetMapping("/oppslag-api/utbetalinger")
     fun getUtbetalingerFraNav(
         @RequestHeader headers: HttpHeaders
-    ): ResponseEntity<List<UtbetalingDto>> {
+    ): ResponseEntity<UtbetalingerResponseDto> {
         val ident = hentFnrFraToken(headers)
         feilService.eventueltLagFeil(ident, "UtbetalingController", "getUtbetalingerFraNav")
         val utbetalinger = utbetalingService.getUtbetalingerFraNav(ident)
+        val response = UtbetalingerResponseDto(utbetalinger, false)
```

Comment:
Burde vi legge inn funksjonalitet for å svare true på at response feilet også?

Reply:
jo, det kan gå an å fikse 👍 feil generert via feilService vil gi samme "utfall" ved behandling av responsen i soknad-api (at driftsinformasjonen blir korrekt)

Reply:
Det var bare en ide ;-) Vi kan styre om vi skal returnere 500 eller true f.eks.

Men dersom det blir det samme i soknad-api, så er det jo ikke så viktig.

Reply:
Se gjerne her hvis du vil vite mer hvordan ting skjer i soknad-api: https://github.com/navikt/sosialhjelp-soknad-api/pull/746 😄